backend/pkg/api/data_access/vdb_summary.go

```diff
@@ -1183,16 +1196,16 @@ func (d *DataAccessService) GetLatestExportedChartTs(ctx context.Context, aggreg
 	var dateColumn string
 	switch aggregation {
 	case enums.IntervalEpoch:
-		table = "view_validator_dashboard_data_epoch_max_ts"
+		table = "validator_dashboard_data_epoch_max_ts"
 		dateColumn = "t"
 	case enums.IntervalHourly:
-		table = "view_validator_dashboard_data_hourly_max_ts"
+		table = "validator_dashboard_data_hourly_max_ts"
 		dateColumn = "t"
 	case enums.IntervalDaily:
-		table = "view_validator_dashboard_data_daily_max_ts"
+		table = "validator_dashboard_data_daily_max_ts"
 		dateColumn = "t"
 	case enums.IntervalWeekly:
-		table = "view_validator_dashboard_data_weekly_max_ts"
+		table = "validator_dashboard_data_weekly_max_ts"
 		dateColumn = "t"
```

Comment:
@sasha-bitfly please revert these changes again, the tables have been renamed in the schema

Reply:
@invis-bitfly [Done](https://github.com/gobitfly/beaconchain/commit/84f1b0431860855be156446ac6335e034be5786c)